hack/e2e/eksctl.sh

```diff
@@ -68,6 +71,7 @@ function eksctl_create_cluster() {
 
   if [[ "$WINDOWS" == true ]]; then
     ${BIN} create nodegroup \
+      --managed=false \
```

Comment:
reading this flag, do we change it to false because this is test nodegroups and we do not want to make it show up from the console?

```
setting --managed=false or using the nodeGroups field creates an unmanaged nodegroup. 
Bear in mind that unmanaged nodegroups do not appear in the EKS console, which as a general rule only knows about EKS-managed nodegroups.
```

Reply:
it's because "Windows is only supported for self-managed (--managed=false flag) nodegroups." (note this is in the WINDOWS clause) . https://eksctl.io/usage/windows-worker-nodes/ . I will add a comment in a follow-up PR for hack/e2e changes